src/vm/eventtrace.cpp

```diff
@@ -319,11 +320,10 @@ ETW::SamplingLog::EtwStackWalkStatus ETW::SamplingLog::SaveCurrentStack(int skip
     {
         return ETW::SamplingLog::UnInitialized;
     }    
-    // The thread should not have a hijack set up or we can't walk the stack. 
+    // The thread should not have a hijack set up or we can't walk the stack.
```

Comment:
Depending on your project policies, this kind of cleanup may belong to a separate "cleanup" commit.